### DISCUSSION TITLE
Send opendtu data to google drive or other cloud

### DISCUSSION THREAD

**Discussion by zigit780** [tbnobody/OpenDTU#2334] [Q&A]
Hi I want to know if there is somebody who uses a solution or knows a solution to send the data from opendtu in google drive or other personal cloud (e.g. magenta cloud) to save it like on a sdcard but not locally but in the cloud. 

And then make a visualization with google sheet maybe or whatever.

Maybe also with a fork which has that functionality.

What I mean that OpenDTU sends direct the data (like also with MQTT) but not on a MQTT-Broker but on a cloud in a file (e.g csv-file or txt-file)

**Comment by stefan123t** [comment]
Thanks but your request probably can be closed as duplicate of #1123.

Please see #11, #176 and especially https://github.com/tbnobody/OpenDTU/issues/1123#issuecomment-2221509521

**Reply by zigit780** [reply]
Thanks I will check that.
